jitterdebugger.c

```diff
@@ -354,6 +378,9 @@ static void *worker(void *arg)
 		}
 	}
 
+	qsort(s->hist_overflows, s->hist_overflows_last,
```

Comment:
Doing the sorting in the measurement thread is not really a good idea.

Reply:
Well doing real work in the measurement thread could be a problem. E.g. sorting could end up using a lot of cycles. I don't like the idea to add to much work to the measurement thread (think about all the cache lines!)